tasks/manage.yml

```diff
@@ -1,5 +1,10 @@
 ---
 
+- name: Adding group user's belong to
```

Comment:
This is a good solution but I don't like to "fix" wrong configurations. I would rather add a `groups` var instead of collecting missing ones in case you have typos or use group names that shouldn't exist. Also using `json_query` requires `jmespath ` to be installed

Reply:
>  I don't like to "fix" wrong configurations.
Well, it could be argued that a "right" configuration should not have redundant data (which could lead to inconsistencies) and then declaring a new group is just putting it into its members. 
But I understand your argument of simplicity and declaring separately the groups is as generally an OS works. 
You could indeed create a ```groups``` var, or explicitely rename your  "default" vars. 
I understand if you don't take the pull request, but by now the bug remains: your ```users_groups``` is the list of default groups as well as the full list of groups, and that's not logic.

Reply:
> I would rather add a `groups` var instead 

`groups` is not a good name for a variable, because with this name ansible builds a structure of the _groups_ declared in the inventory and their hosts as members.